index.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'react-native-toast-message' {
+  import React from 'react';
+  const Toast: {
+    show(options: {
+      type?: 'success' | 'error' | 'info';
+      position?: 'top' | 'bottom';
+      text1: string;
+      text2: string;
```

Comment:
and texts as optional

Reply:
@spidi123q can you make these 2 fields optional as well? Then I can merge it. Thanks!